sound/soc/sof/intel/hda-loader.c

```diff
@@ -416,6 +417,7 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
 	int ret, ret1, i;
 
 	if ((sdev->fw_ready.flags & SOF_IPC_INFO_D3_PERSISTENT) &&
+	    !(sof_debug_check_flag(SOF_DBG_IGNORE_D3_PERSISTENT)) &&
```

Comment:
this patch has to be squashed with the previous one no? Because we'd enable it by default in the previous one and only disable it based on the module param in this one

Reply:
my design here is to enable it by default if the FW claims it supports the feature. So this subsequent module param is only adding an interface to disable using the FW feature on the host side, for debugging purpose only.